consensus/conf.bzl

```diff
@@ -7,6 +8,7 @@ CONSENSUS_MAINNET_VARIABLES = [
 ]
 
 CONSENSUS_TESTNET_VARIABLES = [
+    "DB_PATH='\"ciri/ciri.db\"'",
```

Comment:
And this one `ciri/ciri-testnet.db` ?

Reply:
Should I also duplicate a database file for this one?